Makefile

```diff
@@ -32,6 +32,30 @@ bin $(BIN): $(GORELEASER)
 	@echo "--- bin ---"
 	@$(GORELEASER) build --snapshot --single-target --rm-dist
 
+# Requires `wixl` from msitools https://wiki.gnome.org/msitools (or `brew install msitools`)
+# If Windows, you can download from here https://github.com/wixtoolset/wix3/releases
+WIN_BIN := dist/func-e_windows_amd64
+WIN_BIN_EXE := $(WIN_BIN)/func-e.exe
+
+$(WIN_BIN_EXE): $(GORELEASER)
+	@echo "--- win-bin ---"
+	@GOOS=windows GOARCH=amd64 $(GORELEASER) build --snapshot --single-target --rm-dist
+
+.PHONY: msi
+msi: $(WIN_BIN_EXE)
+ifeq ($(OS),Windows_NT)  # same for Windows 10 etc
```

Comment:
@mathetake earlier I was trying to explain that how in windows it is two commands to accomplish what in linux/darwin is one. This is the thing I referred to.

Hover goes a bit farther and does this and some other bits in go... not quite sure want to go there at this point, but we could certainly optimize later. https://github.com/go-flutter-desktop/hover/blob/9f74033acaeabec51906e9542d8706111b98fe31/cmd/packaging/windows-msi.go

Main thing is that make is already doing the release, so probably good enough to leave this as an `ifeq`